### PR TITLE
Update crawlers after site changes

### PR DESCRIPTION
**Pull request by datagutten**

### CODE REVIEW

File: comics/comics/perrybiblefellowship.py [hunk 1]
```diff
@@ -15,12 +15,14 @@ class Crawler(CrawlerBase):
...
 
     def crawl(self, pub_date):
-        feed = self.parse_feed("http://www.pbfcomics.com/feed/feed.xml")
+        feed = self.parse_feed("https://pbfcomics.com/feed/")
         for entry in feed.for_date(pub_date):
             page = self.parse_page(entry.link)
             images = page.src("div#comic img", allow_multiple=True)
             crawler_images = []
             for image in images:
+                if image[0:4] != 'http':
```
Comment (jodal) [inline comics/comics/perrybiblefellowship.py]: `if not image.startswith("http"):`

File: comics/comics/questionablecontent.py [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
-        if text:
-            text = re.sub(r"\s{2,}", "\n\n", text).strip()
-        return CrawlerImage(url, title, text)
+        feed = self.parse_feed("https://www.questionablecontent.net/QCRSS.xml")
+        for entry in feed.for_date(pub_date):
+            title = entry.title
+            url = entry.summary.src('img[src*="/comics/"]')
+            text = entry.summary.text("p", allow_multiple=True)
+            if len(text) > 1:
+                text = text[1]
```
Comment (jodal) [inline comics/comics/questionablecontent.py:28]: I guess this should be `text[0]`? If not, you might be out of bounds.

File: comics/comics/anleggsplassen.py [hunk 3]
```diff
@@ -1,40 +1,43 @@
...
+
 from comics.aggregator.crawler import CrawlerBase, CrawlerImage
 from comics.core.comic_data import ComicDataBase
 
 
 class ComicData(ComicDataBase):
     name = "Anleggsplassen"
     language = "no"
-    url = "https://www.at.no/"
+    url = "https://www.at.no"
```
Comment (jodal) [inline comics/comics/anleggsplassen.py:11]: I see no reason to remove the slash here. Most other `url`s have it.

File: comics/comics/anleggsplassen.py [hunk 4]
```diff
@@ -1,40 +1,43 @@
 import re
 
+import dateutil.parser
```
Comment (jodal) [inline comics/comics/anleggsplassen.py]: `dateutil` is only a transitive dependency for now, so if you really need this, it must be added to the dependencies with:

```
poetry add dateutil
```

File: comics/comics/anleggsplassen.py [hunk 5]
```diff
@@ -30,7 +29,7 @@ def crawl(self, pub_date):
             date_string = article_page.content(
                 'meta[property="article:published_time"]'
             )
-            date = dateutil.parser.parse(date_string).date()
+            date = datetime.strptime(date_string[:10], "%Y-%m-%d").date()
```
Comment (jodal) [inline comics/comics/anleggsplassen.py:32]: You can use `date = self.string_to_date(date_string[:10], "%Y-%m-%d")` and avoid the import.

**Comment by datagutten** [comment]
I forgot this, and some of the changes are now out of date, I will make a new pull request later.

**Comment by jodal** [comment]
Lovely! I'll be around to review and merge during Xmas :-)

**Pull request closed without merging**